package Pro;
message ServerStopS2C{}